backend/src/main/java/com/htecgroup/skynest/filter/CustomAuthorizationFilter.java

```diff
@@ -43,7 +43,13 @@ protected void doFilterInternal(
         return;
       }
 
-      String authorizationHeader = request.getHeader(JwtUtils.AUTH_HEADER);
+      String authorizationHeader;
+      if (request.getServletPath().startsWith(UrlUtil.REFRESH_TOKEN)) {
+        authorizationHeader = request.getHeader(JwtUtils.REFRESH_TOKEN_HEADER);
```

Comment:
Why ?

Reply:
It was so the Authorization goes through Refresh token when the refresh endpoint is called, because the access token will be expired then, but according to your previous comment, I'm changing it now.